continuous_integration/gpuci/build.sh

```diff
@@ -41,7 +41,7 @@ gpuci_logger "Install distributed"
 python -m pip install git+https://github.com/dask/distributed
 
 gpuci_logger "Install dask"
-python setup.py install
+python -m pip install --quiet --no-deps -e .
```

Comment:
Not sure how noisy this is, but I can imagine wanting this output in the future for debugging (just a hunch) 

```suggestion
python -m pip install --no-deps -e .
```

Reply:
Why can we use `--no-deps`? Are those being installed in the base image or somewhere else?

Reply:
Yeah, the gpuCI conda environment is the Dask python 3.9 CI environment with a few RAPIDS packages installed on top ([Dockerfile](https://github.com/rapidsai/dask-build-environment/blob/dfa14787ca32e96cb22c3c46352f4bfa1d0536c2/dask/Dockerfile#L13) for reference).